ripe/atlas/tools/helpers/validators.py

```diff
@@ -50,6 +51,29 @@ def ip_or_domain(string):
 
         return string
 
+    @classmethod
+    def comma_separated_integers_or_file(cls, string):
+
```

Comment:
a one line comment stating that it's handling also stdin would be nice, since function name is mentioning only file.